code/Coding_Abgabe_I/ÜBERUNS.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <link rel="stylesheet" href="stylesheet.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<div id="mySidenav" class="sidenav">
+    <a href="./INDEX.html" id="HOME">HOME</a>
+    <a href="./IDEE.html" id="IDEE">IDEE</a>
+    <a href="./FRAGEBOGEN.html" id="FRAGEBOGEN">FRAGEBOGEN</a>
+    <a href="./ÜBERUNS.html" id="ÜBER">ÜBER UNS</a>
+  </div>
+
+<body>
+    <div class="über">
+        <h2>ÜBER HÄSCHGWÜSST</h2>
+        <p>häschgwüsst  <a href="https://www.instagram.com/haeschgwuesst/" class="fa fa-instagram"></a>
+            – Informationen, Fakten, Geschichten – Dinge, die wir ALLE wissen müssen.
+            häschgwüsst sind Nina, Justine und Roberta. Gemeinsam haben wir 2019 die Vagina Monologe nach Zürich gebracht. 
+            Daraus ist häschgwüsst entstanden. 
+            Wichtige Fakten und spannende Geschichten rund um die Themen Feminismus, Gleichstellung und Weltgeschehen grafisch aufbereitet.
+        </p>
+       
+</div>
+
+<div class="allwomen">
+<div class="woman1">👩🏻 <a href="./ju.html" id="Justine">Justine</a></div>
+<div class= "woman2">👩🏼 <a href="./ni.html" id="Nina">Nina</a></div>
+<div class= "woman3">👧🏼<a href="./ro.html" id="Roberta">Roberta</a></div>
+</div>

```

Comment:
It looks nice with the circles, I like the idea! 

Here I would suggest using flex-box in CSS. You can for example use the name:
`<div class="woman">` for each of the three women and keep the name "allwomen" for the wrapping div or alt. something like `<div class="women-main">` and in CSS for .women-main to use: 
`display: grid; `
`flex-direction: row;`

I recommend:
https://css-tricks.com/snippets/css/a-guide-to-flexbox/